todos/src/reducers/todoReducers.js

```diff
@@ -36,7 +37,7 @@ const todoReducers = (state = initialData, action) => {
             ...todo,
             isCompleted: action.isCompleted,
             date: action.date,
-            completedDate: action.completedDate,
+            completedDate: new Date().toLocaleDateString(),
```

Comment:
Reducers are what called pure functions. They must be deterministic in nature, meaning for each set of input n, it should always return the same result m.

We should always try to avoid doing things that are non deterministic in nature such as calling API, random functions, date functions inside reducers. 

More [here](https://redux.js.org/style-guide/#reducers-must-not-have-side-effects)